pkg/sysfs/system.go

```diff
@@ -891,7 +891,6 @@ func (n *node) MemoryInfo() (*MemInfo, error) {
 		map[string]interface{}{
 			"MemTotal:": &buf.MemTotal,
 			"MemFree:":  &buf.MemFree,
-			"MemUsed:":  &buf.MemUsed,
```

Comment:
Why do we remove this? It is not obvious from the commit.

Reply:
Because I fsck'd up the commit...

Reply:
Updated with a hopefully less b0rken fix.